pkg/kubelet/pod_workers.go

```diff
@@ -291,9 +291,9 @@ func killPodNow(podWorkers PodWorkers, recorder record.EventRecorder) eviction.K
 		}
 
 		// we timeout and return an error if we don't get a callback within a reasonable time.
-		// the default timeout is relative to the grace period (we settle on 2s to wait for kubelet->runtime traffic to complete in sigkill)
+		// the default timeout is relative to the grace period (we settle on 3s to wait for kubelet->runtime traffic to complete in sigkill)
```

Comment:
How is this time period determined? What is it based on? Do we wait for the the runtime kill to return or do we wait for a the status to be terminated from the runtimes?